faker/providers/internet/__init__.py

```diff
@@ -115,6 +115,13 @@ class Provider(BaseProvider):
         '?txtsize=55&txt={width}x{height}&w={width}&h={height}',
         'https://www.lorempixel.com/{width}/{height}',
         'https://dummyimage.com/{width}x{height}',
+        'https://baconmockup.com/{width}/{height}',
```

Comment:
I am a lover of bacon myself, but I'm afraid the images returned by this service might be too meaty, and I'd like to avoid any issue related to content. Could you remove it?

Reply:
Sure. That's why we have code review. :)